source/Core/Threads/UI/logic/Soldering.cpp

```diff
@@ -9,12 +9,16 @@
 OperatingMode handleSolderingButtons(const ButtonState buttons, guiContext *cxt) {
   if (cxt->scratch_state.state1 >= 2) {
```

Comment:
What about:
```suggestion
  if (cxt->scratch_state.state1 > 1) {
```

Reply:
Since 0-3 are defined states, the former is easier to read for me. But this is a personal preference.

Reply:
It's just a suggestion. 😊